include/revng/Storage/Path.h

```diff
@@ -0,0 +1,93 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "llvm/ADT/SmallString.h"
+#include "llvm/Support/Path.h"
+
+#include "revng/Storage/StorageClient.h"
+#include "revng/Support/Debug.h"
+
+namespace revng {
+
+class Path {
+private:
+  revng::StorageClient *Client;
+  std::string StoredPath;
+
+public:
+  explicit Path(StorageClient *Client, const llvm::StringRef StoredPath) :
+    Client(Client), StoredPath(StoredPath.str()){};
+
+  static Path fromLocalStorage(llvm::StringRef Path);
+
+public:
+  std::string toString() const debug_function {
```

Comment:
This looks like `.dump`, not `toString`.

Reply:
renamed to `.dump`